tests/keyword-detection.test.html

```diff
@@ -0,0 +1,38 @@
+<html>
+  <body>
+    <div data-ea-publisher="test"></div>
+    <main>
+      <p>
+        PyTorch is an important module for machine learning. PyTorch can use
+        your GPU to crunch numbers faster than a CPU.
+      </p>
+    </main>
+
+    <script type="module">
+      import { expect } from "@open-wc/testing";
+      import { runTests } from "@web/test-runner-mocha";
+      import { default as sinon } from "sinon";
+
+      import { wait, detectedKeywords, Placement } from "../index";
+
+      // Don't actually call the server
+      // https://sinonjs.org/releases/v17/stubs/
+      let stub = sinon.stub(Placement.prototype, "fetch");
+      const response_html =
+        "<div><!-- A real ad would be here normally --></div>";
+      const elem_placement = document.createElement("div");
+      elem_placement.innerHTML = response_html;
+      stub.resolves(elem_placement.firstChild);
+
+      runTests(async () => {
+        describe("EthicalAds library", () => {
+          it("keyword detection", async () => {
```

Comment:
I should have added this to all the test files, but I'm also not certain how important it is. Sinon mentions this is needed to avoid memory leaks in the test suite, but maybe it also does more than that?

```suggestion
      runTests(async () => {
        beforeEach(() => {
          stub.reset();
        });
        describe("EthicalAds library", () => {
          it("keyword detection", async () => {
```

I'm guessing it is not a big deal either way, so just noting this for later.

Reply:
It caused problems when there were multiple tests in the file (as in `missing-placement.test.html`). Perhaps we'll need to figure that out but it appears to work without it.